doctor/src/index.js

```diff
@@ -499,7 +495,7 @@ aggregateCrawlerInfo.then(function (indices) {
             "attachments": reports
         };
 
-        slack = new Slack(slackHook);
+        var slack = new Slack(slackHook);
```

Comment:
Why changing it if you don't change the way you use it?

Reply:
because it's more correct 😄